### ISSUE TITLE
Polohovací platforma

### ISSUE THREAD

**Issue by roman-dvorak** [AstronomickaExpedice/NaturalPlanetarium#1]
Lase přirodního planetária bude umístěna na platformě (montáži) v režimu ALT-AZ. Montáž by měla být ovládána pomocí dvojice krokových motorů [NEMA17](https://reprap.org/wiki/File:Step_motor_nema_17_stepper_motor.jpg).

Stručný nákres:
![20200402_010807](https://user-images.githubusercontent.com/5196729/78195428-de737a80-747f-11ea-8f0d-3bb9b41067a0.jpg)

Celé se to bude skládat z několika tištěných dílů:
- [ ] Spodní podložka: Bude držet motor a AZ osu dohromady. Může obsahovat nějaké nožičky
- [ ] AZ řemenice
- [ ] AZ část: První pohyblivá část. Může být sestavena ze dvou/tří častí. Pro optimalizaci tisku. Z jedné strany bude příruba motoru. Je potřeba počítat s tím, že na horní čast povedou nějaké dráty atd.. Je pro ně potřeba vytvořit místo, průchod, .. 
- [ ] ALT řemenice
- [ ] ALT podložka, držák laseru: část, ke které bude připevněn samotný laser. Bude se otáčet jak v ALT tak AZ směru. 
- [ ] Řemenice na motory

Pro generování řemenic lze použít tuto knihovnu: https://www.thingiverse.com/thing:16627

**Comment by roman-dvorak** [comment]
Celé by to šlo zmenšit a zjednodušit následovně. Tento hranatý tvar nemusí být vůbec dodržen a může být nějak optimalizován pro úsporu materiálu/rozměrů či jiných důvodů.

![20200402_014536](https://user-images.githubusercontent.com/5196729/78196822-dddce300-7483-11ea-96d2-26c66f1ef1ff.jpg)

**Comment by IvetaL** [comment]
Prvni navrh 
![91890912_689281041844640_1113671559425294336_n](https://user-images.githubusercontent.com/62916335/78284893-572a1380-7520-11ea-9b23-1ee56ae971d9.jpg)
![92154528_2609765105973063_1198833317726650368_n](https://user-images.githubusercontent.com/62916335/78284902-58f3d700-7520-11ea-84c4-89dc1e51ee9c.jpg)

**Comment by roman-dvorak** [comment]
Tohle řešení umístění motoru pro ALT osu se mi docela líbí. Vyřeší se tím častečně vyvážení oproti laseru na AZ osu.. Motor bych však otočil obráceně. Aby jeho osa směřovala do středu a ALT osa nemusela být tak dlouhá. ALT řemenice by následně byla hned u té střední svislé stěny. 

Když se na to dívám, tak by bylo možné sloučit spodní (AZ) řemenici s následujícím dílem, který bude držet ALT motor. 

Možná by to chtělo rozkreslit ještě detaily uložení jednotlivých os a ložisek použitích v těch osách.

**Comment by IvetaL** [comment]
PRVNI NAVRH 
 
PUDORYS
![Planetarium1](https://user-images.githubusercontent.com/62916335/78457592-abadca00-76ab-11ea-9c36-8884c4b889a7.png)

NARYS
![Planetarium2](https://user-images.githubusercontent.com/62916335/78457593-ad778d80-76ab-11ea-8983-426f7b76c980.png)

BOKORYS
![Planetarium3](https://user-images.githubusercontent.com/62916335/78457594-ae102400-76ab-11ea-8e0a-b464d6aa0096.png)

BOKORYS
![Planetarium4](https://user-images.githubusercontent.com/62916335/78457595-aea8ba80-76ab-11ea-93f0-481ecbcd70ea.png)


![Planetarium8](https://user-images.githubusercontent.com/62916335/78457600-b0727e00-76ab-11ea-817f-5f01ddeb97e3.png)

![Planetarium5](https://user-images.githubusercontent.com/62916335/78457596-af415100-76ab-11ea-9e23-7a82bd70b812.png)

![Planetarium6](https://user-images.githubusercontent.com/62916335/78457597-afd9e780-76ab-11ea-8309-dd4be1edddbd.png)

![Planetarium7](https://user-images.githubusercontent.com/62916335/78457598-afd9e780-76ab-11ea-9fcb-ae1675ad5889.png)

**Comment by roman-dvorak** [comment]
Jo, super napad si to takhle rozvrhnout! :) Je tam videt mnoho detailu. Asi by bylo dobre Ti napsat o knihovne [NopSCADlib](https://github.com/nophead/NopSCADlib), ktera obsahuje modely krokovych motoru, lozisek a podobnych (uzitecnych) veci, abys to nemusea vzdy modelovat znovu. Na te strance jsou obrazky veci, ktere to podporuje a umi modelovat. Tak se tam zkus podivat. 

Napriklad to usporadani ALT motoru je o dost lepsi, nez jsem navrhoval v povodnim nakresu. Je potreba si dat pozor, aby tam nekde nekolidovaly nejake rozmery treba prave s temi motory. Bojim se predevsim o to, aby tam nekde nenarazel laser do obou motoru. To se ale vyresi zvetsenim rozestupu. Coz predpokladam, ze je nejak parametricky? :)

**Comment by kaklik** [comment]
To vypadá dobře! :)  Myslím že bych pro to našel pak i další uplatnění, kromě venkovního planetária. Taktéž jsem teď chtěl napsat o NopSCADlib, ale vidim, že mě předběhl Roman. :)

**Comment by IvetaL** [comment]
Na knihovnu se podivam, diky! Aby laser nenarazil do AZ motoru je osetrene parametrem a dole mam jeste kontrolu. U toho ALT motoru jsem kontrolovala ze to taky sedi, ale jeste to myslim v poslednim nahranem souboru zparametrizovane neni, ale ted uz to tam mam. Ale vetsina tech rozestupu jde taky zvetsit parametricky, vetsinou mam zatim zadane 5 nebo 10mm.
Bude to jeste vse o trosicku vetsi az tam pridam ulozeni laseru a motoru, ale o moc ne.
